### ISSUE TITLE
Accessibility on Quick Start Page

### ISSUE THREAD

**Issue by samilieberman** [reactjs/reactjs.org#8594]
On the [Quick Start page](https://react.dev/learn), there is an accessibility issue that should be fixed to `Ensure the order of headings is semantically correct`. The axe chrome extension can be used to validate the fix. More info below.

<img width="1727" height="646" alt="Image" src="https://github.com/user-attachments/assets/af298d2a-48fd-4883-baa2-5a3abab8ee2a" />

<img width="603" height="589" alt="Image" src="https://github.com/user-attachments/assets/9c0c7c1a-346c-420d-93fa-533de8d76ade" />

**Comment by taauntik** [comment]
Checking ^
